llvm/test/Transforms/LoopVectorize/check-no-vectorize.ll

```diff
@@ -0,0 +1,36 @@
+; This test asserts that we don't emit both
```

Comment:
Nit: I'd probably name this test dbg-print-no-vectorize.

Reply:
I think the name is fine.

Reply:
Reflow comment
```suggestion
; This test checks that we don't emit both successful and unsuccessful
```